packages/vue/src/components/field/field-select.vue

```diff
@@ -8,17 +8,28 @@ export interface FieldSelectProps
     /**
      * @vue-ignore
      */
-    SelectHTMLAttributes {}
+    Omit<SelectHTMLAttributes, 'value'> {
+  modelValue?: SelectHTMLAttributes['value']
+}
 </script>
 
 <script setup lang="ts">
 import { ark } from '../factory'
 import { useFieldContext } from './use-field-context'
 
-defineProps<FieldSelectProps>()
+const props = defineProps<FieldSelectProps & { modelValue?: string }>()
 const field = useFieldContext()
+
+const emit = defineEmits(['update:modelValue'])
 </script>
 
 <template>
-  <ark.select v-bind="field.getSelectProps()" :as-child="asChild"><slot /></ark.select>
+  <ark.select
+    v-bind="field.getSelectProps()"
+    :value="props.modelValue"
+    @change="(event) => emit('update:modelValue', (event.target as HTMLSelectElement).value)"
+    :as-child="asChild"
```

Comment:
If the v-bind name and the value name is the same, it's now (since v3.4 of Vue) not needed to type them both. It can just be `:as-child`.

```
<!-- same-name shorthand (3.4+), expands to :src="src" -->
<img :src />
```

Reply:
That was already there. But I can go ahead and update it here since it's small 😁